dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/storage/impl/S3Utils.java

```diff
@@ -171,6 +183,26 @@ public String getResourceFileName(String tenantCode, String fileName) {
         return String.format(FORMAT_S_S, getS3ResDir(tenantCode), fileName);
     }
 
+    @Override
+    public String getResourceFileName(String fullName) {
```

Comment:
It seems the same method `getResourceFileName` in `HadoopUtils`?

Reply:
> It seems the same method `getResourceFileName` in `HadoopUtils`?

ya for now, it is. 

In the future release, I should come up with a better way to get file name.